test/Microsoft.OpenApi.Hidi.Tests/Microsoft.OpenApi.Hidi.Tests.csproj

```
@@ -25,39 +25,12 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Update="UtilityFiles\OpenApiDocumentMock.cs">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </Compile>
-  </ItemGroup>
-
-  <ItemGroup>
-    <None Update="UtilityFiles\appsettingstest.json">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-    <None Update="UtilityFiles\postmanCollection_ver1.json">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-    <None Update="UtilityFiles\postmanCollection_ver2.json">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-    <None Update="UtilityFiles\postmanCollection_ver3.json">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-    <None Update="UtilityFiles\postmanCollection_ver4.json">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-    <None Update="UtilityFiles\SampleOpenApi.yml">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Update="UtilityFiles\Todo.xml">
+    <None Update="UtilityFiles\*.*">
```

This is also going to copy the mock CS file.
We could either move it to a separate folder or do this I guess

```suggestion
    <None Update="UtilityFiles\*.(json|yml|xml)">
```